sycl/test/basic_tests/image/image_write.cpp

```diff
@@ -0,0 +1,276 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_BE=%sycl_be %t.out
+
+/**
+ * Tests clamp_to_edge addressing mode with nearest and linear filtering modes
+ * on a 4x4 image.
+ *
+ * Expected addressing mode and filtering results are given by the algorithm in
+ * the OpenCL 1.2 specification, Section 8. Image Addressing and Filtering
+ *
+ * https://www.khronos.org/registry/OpenCL/specs/opencl-1.2.pdf#page=329
+ *
+ * Confirmed to pass with -fsycl-targets=spir64-unknown-linux-sycldevice
+ *
+ * Enable sampled test with -DSAMPLED_TEST
+ *
+ * Enabled unsampled tests with -DUNSAMPLED_TEST
+ *
+ * When using the sampled image
+ * Compilation currently fails for -fsycl-targets=nvptx64-nvidia-cuda-sycldevice
```

Comment:
Could you XFAIL:cuda the test as soon as it is expectedly fail on cuda? 
Also it may fail on configuration without neither opencl, nor level0 nor cuda RTs available.

Reply:
These comments are outdated. I have removed them.